App/BraveWidgets/FavoritesWidget.swift

```diff
@@ -138,15 +139,8 @@ private struct FavoritesGridView: View {
             label: {
               Group {
                 if let attributes = favorite.favicon, let image = attributes.image {
-                  FaviconImage(image: image, contentMode: attributes.contentMode, includePadding: {
-                    switch attributes.contentMode {
-                    case .scaleToFill, .scaleAspectFit, .scaleAspectFill:
-                      return false
-                    default:
-                      return true
-                    }
-                  }())
-                    .background(Color(attributes.backgroundColor ?? .clear))
+                  FaviconImage(image: image, contentMode: .scaleAspectFit, includePadding: false)
```

Comment:
I believe we dont use contentMode in FavIcon object anymore and we will always pass . scaleAspectFit to FavIocnImage?

I think we can make `contentMode` parameter defaulted to .scaleAspectFit in this case ?

Reply:
This is a SwiftUI struct. I left it in case we do use it for scaling the SwiftUI image.